packages/itmat-ui-react/src/components/login/resetPasswordPage.tsx

```diff
@@ -17,6 +18,18 @@ export const ResetPasswordPage: React.FunctionComponent<ResetPasswordPageProps>
 
     const history = useHistory();
     const [passwordSuccessfullyChanged, setPasswordSuccessfullyChanged] = React.useState(false);
+    const { loading, error } = useQuery(GQLRequests.VALIDATE_RESET_PASSWORD, { variables: {
+        encryptedEmail: encryptedEmail,
+        token: token
+    }});
+    if (loading) { return <LoadSpinner />; }
+    if (error) {
+        return <div className={css.login_wrapper}>
+            <div className={css.login_box}>
+                <h1>The link is invalid. Please make a new request.</h1>
+            </div>
```

Comment:
Could we add a button permitting return to the login page. Something like :
```jsx
<Button onClick={() => {
    history.push('/');
}}>
    Go back to login
</Button>
```

Reply:
Sure!